crates/viewer/re_web_viewer_server/src/lib.rs

```diff
@@ -169,6 +170,12 @@ impl WebViewerServer {
             thread_handle.join().ok();
         }
     }
+
+    /// Keeps the web viewer running until the parent process shuts down.
+    pub fn detach(self) {
+        #[expect(clippy::mem_forget)]
+        std::mem::forget(self);
```

Comment:
I'm pretty sure this will work fine, but if the thread shuts down for some reason, we will leak memory, which is unfortunate. If we want to avoid joining here, we can do this instead:
```suggestion
        let Self { inner: _, thread_handle } = self;
        // dropping the thread_handle detaches the thread.
        drop(thread_handle);
```